zip-0230.rst

```diff
@@ -4,11 +4,347 @@
+|                             |                          |                                      |spendAuthSigsOrchard corresponding to this Action.          |
++-----------------------------+--------------------------+--------------------------------------+------------------------------------------------------------+
+|``32``                       |``cmx``                   |``byte[32]``                          |The x-coordinate of the note commitment for the output note.|
++-----------------------------+--------------------------+--------------------------------------+------------------------------------------------------------+
+|``32``                       |``ephemeralKey``          |``byte[32]``                          |An encoding of an ephemeral Pallas public key               |
++-----------------------------+--------------------------+--------------------------------------+------------------------------------------------------------+
+|``612``                      |``encCiphertext``         |``byte[580]``                         |The encrypted contents of the note plaintext.               |
++-----------------------------+--------------------------+--------------------------------------+------------------------------------------------------------+
+|``80``                       |``outCiphertext``         |``byte[80]``                          |The encrypted contents of the byte string created by        |
+|                             |                          |                                      |concatenation of the transmission key with the ephemeral    |
+|                             |                          |                                      |secret key.                                                 |
++-----------------------------+--------------------------+--------------------------------------+------------------------------------------------------------+
+
+The encodings of each of these elements are defined in §7.5 ‘Action Description Encoding
+and Consensus’ of the Zcash Protocol Specification [#protocol-actiondesc]_.
+
+Orchard-ZSA Asset Burn Description
+----------------------------------
+
+An Orchard-ZSA Asset Burn description is encoded in a transaction as an instance of an ``AssetBurn`` type:
+
++-------+---------------+-----------------------------+---------------------------------------------------------------------------------------------------------------------------+
+| Bytes | Name          | Data Type                   | Description                                                                                                               |
++=======+===============+=============================+===========================================================================================================================+
+| 32    | ``AssetBase`` | ``byte[32]``                | For the Orchard-based ZSA protocol, this is the encoding of the Asset Base :math:`\mathsf{AssetBase}^{\mathsf{Orchard}}`. |
```

Comment:
This should use $\mathsf{repr}_{\mathbb{P}}$. Non-blocking.

Reply:
Updated this to point correctly to the byte encoding that had already been defined in ZIP 226.